README.md

```diff
@@ -5,12 +5,40 @@
 
 A general/permissive SQL grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 
-## Notes
+## Installation
 
-We don't commit the generated parser files to the main branch. Instead, you can find them on the
-[gh-pages](https://github.com/DerekStride/tree-sitter-sql/tree/gh-pages) branch. They are also hosted on the [GitHub
-pages site](https://derek.stride.host/tree-sitter-sql/). We're open to feedback & encourage you to [open an
-issue](https://github.com/DerekStride/tree-sitter-sql/issues/new) to discuss any problems.
+### Step 1: Download the parser files
+
+**Using `git`**
+```bash
+git clone https://github.com/DerekStride/tree-sitter-sql.git
+cd tree-sitter-sql
+git checkout gh-pages
+```
+
+**Using `curl`**
+```bash
+curl -LO https://github.com/DerekStride/tree-sitter-sql/archive/refs/heads/gh-pages.tar.gz
+tar -xzf gh-pages.tar.gz
+cd tree-sitter-sql-gh-pages
+```
+
+**Note:** We don't commit the generated parser files to the main branch. Instead, you can find them on the
```

Comment:
Maybe change the order of `Notes` and `Installation` since the reasoning why we have the gh-pages branch is written in `Notes`